packages/office-ui-fabric-react/src/components/SwatchColorPicker/SwatchColorPicker.scss

```diff
@@ -52,14 +60,25 @@
   }
 }
 
-.cell {
+.cell, .circleCell {
   padding: 0px;
   overflow: visible;
 
+  &:focus {
+    width: 40px;
+  }
+
+  &:hover .svg,
+  &:focus .svg {
```

Comment:
I'd only show focus if someone is using keyboard. You can use the focus mixin to only display focus when it has been gained through keyboarding around.  That way if focus is given automatically, like when a panel is started, it doesn't look focused until a user activates it. @phkuo correct me if I'm wrong
```
@include focus(true) {
    border: 4px solid $ms-color-neutralQuaternaryAlt
}

Reply:
I didn't know about the focus mixin, I can look into using it. The border property isn't for focus though, it's for hover - the focus outline is set using the box-shadow property down in &:focus::after

Reply:
Hmm okay.

Reply:
using focus() mixin now down below